### PR TITLE
Recent Works and About Sections

### PR DESCRIPTION
**Pull request by lucabenaventew**
# Project Updates :green_circle: :green_circle: :green_circle: 

- [x] Used CSS GRID and FLEXBOX COMBINED to create the sections
- [x] Used HTML and CSS best practices
- [x] Implemented the required ul and li for categories in "Work" and "About myself"
- [x] Implemented button interactions

### CODE REVIEW

**Comment by lucabenaventew** [comment]
Good morning, the stylint linter give me errors but if I fix them it will move all my organized code and I will confuse everytime after this. I prefer to leave my CSS code as it is because its more clear for me

File: Home/index.html [hunk 1]
```diff
@@ -47,18 +55,192 @@
...
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>
+            </figcaption>
+        </figure>
+        <div class="works_grid_container">
+            <div class="card_hovered">
+                <button type="button">See Project</button>
+            </div>
+
```
Comment (jaflih) [inline Home/index.html:97]: Please fix the bug in this card:
it overlaps the previous section

image

File: CSS/3-recent-works.css [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+    background-image: url("../img/bg/secondbg.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+  
+  .recent_works_container .works_grid_container div:nth-child(2) {
+    background-image: url("../img/bg/bg2.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
```
Comment (jaflih) [inline CSS/3-recent-works.css:133]: Please correct the design to stick to Figma.
Here it lake the button "see project" and the design "HTML" "bootstrap" "ruby" is not the same as in Figma

image

File: CSS/4-about.css [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+.about {
+    background-color: #1c1a19;
+    background-image: url(../img/bg/Frame.png);
+    background-repeat: no-repeat;
+    background-position: 100% 10%;
+    background-size: 40%;
+    display: grid;
+    grid-template-columns: 10% 40% 40% 10%;
+    grid-template-rows: 1fr 10% 1fr;
+  }
```
Comment (jaflih) [inline CSS/4-about.css:12]: Please stick to Figma, please reduce the margin-top

image

File: Home/index.html [hunk 4]
```diff
@@ -47,18 +55,192 @@
...
+                        Skills
+                    </h4>
+                </div>
+                <ul>
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
```
Comment (jaflih) [inline Home/index.html:240]: Good job for using ul/li for listing the specificity like Codekit, Codepen ... 👍 

For the same reason, you have to use ul/li for listing these 3 elements : Languages ,Frameworks ,Skills

**Review by jaflih — CHANGES_REQUESTED** [review]
Hi @lucabenaventew , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✔️ Linters passed 💯
✔️ The content is aligned with flexbox and grid 🥇 
✔️ Great commit history

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @jaflih in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lucabenaventew** [comment]
Please I already make everything I could, I'm running out of time, please kindly accept my code and I will improve my code during tomorrow, everything is okay, I just have to make little changes, regards 
Luca Benavente

File: CSS/3-recent-works.css [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+    font-size: 20px;
+    line-height: 28px;
+    color: #091e42;
+  }
+  
+  .recent_works_container figcaption ul {
+    display: flex;
+    gap: 1rem;
+  }
+
```
Comment (uwadonat) [inline CSS/3-recent-works.css:72]: - [ ] AS mentioned by the previous reviewer some of your feature are not matching the given design. kindly use [grid](https://www.w3schools.com/css/css_grid.asp) and [flex box ](https://www.w3schools.com/css/css3_flexbox.asp)to fix this.

here is your design
Screenshot from 2022-05-28 23-52-39

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **[lucabenaventew](https://github.com/lucabenaventew)**, :hand: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

### To highlight! :+1: 
- Your commit messages are very descriptive ✔️
- No linter errors ✔️

 ## Status: Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: CSS/3-recent-works.css (hunk 5 above)
Comment (lucabenaventew) [reply CSS/3-recent-works.css:72]: I have already completed all the requirements, so what are your requirements? please clarify

**Review by HillaryOkello — APPROVED** [review]
# Hi @lucabenaventew,

## To Highlight
- Descriptive Pull request ✅ 
- Your commit messages are descriptive ✅ 
- No linter errors ✅ 

Nice work implementing the changes required by the previous reviewer 👍🏻 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**